### PR TITLE
findRachaByOwner

### PR DESCRIPTION
**Pull request by JPedro-M**
Added endpoint to find rachas created by owner

### CODE REVIEW

File: src/main/java/ifsul/agileproject/rachadinha/controller/RachaController.java [hunk 1]
```diff
@@ -18,16 +24,25 @@
...
-    //Criar racha
-    @PostMapping("/criar")
-    public ResponseEntity<RachaResponseDTO> createRacha(@RequestBody RachaRegisterDTO rachaDTO) {
-        Racha racha = rachaService.saveRacha(rachaDTO);
+		return new ResponseEntity<>(rachaResponseDTO, HttpStatus.CREATED);
+	}
 
-        RachaResponseDTO rachaResponseDTO = RachaResponseDTO.transformarEmDto(racha);
+	@GetMapping("/findByOwner/{id}")
+	public ResponseEntity<List<Racha>> findByOwner(@PathVariable Long id) {
```
Comment (edukatiau) [inline src/main/java/ifsul/agileproject/rachadinha/controller/RachaController.java:40]: Maybe is better return a List of RachaRespondeDTO

**Review by edukatiau — CHANGES_REQUESTED** [review]
Very well, just do a check

File: src/main/java/ifsul/agileproject/rachadinha/controller/RachaController.java (hunk 1 above)
Comment (JPedro-M) [reply src/main/java/ifsul/agileproject/rachadinha/controller/RachaController.java]: Return type changed to RachaResponseDTO

**Review by edukatiau — APPROVED** [review]
Nice

**Pull request merged**